python/tests/variant_tests/cpmx_variant.py

```diff
@@ -0,0 +1,115 @@
+from tests.aggregate_tests.aggtst_base import TstView, TstTable
+
+
+# Complex Types(Array, Map, Row)
+class varnttst_cmpx_var_tbl(TstTable):
+    """Define the table used by the complex type tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TABLE varnt_cmpx_var_tbl(
+                      id INT,
+                      arr VARIANT ARRAY,
+                      mapp MAP<VARCHAR, VARIANT>,
+                      roww ROW(v1 VARIANT, v2 VARIANT)
+                      )"""
+        self.data = [
+            {
+                "id": 0,
+                "arr": ["12, 22", "hello, hi, bye"],
+                "mapp": {"a": "ciao!", "b": "hello 45 234"},
+                "roww": {"v1": "20", "v2": "bye bye!"},
+            },
+            {"id": 1, "arr": None, "map": None, "roww": None},
+            {
+                "id": 2,
+                "arr": [None, None],
+                "map": {"a": None, "b": None},
+                "roww": {"v1": None, "v2": None},
+            },
+        ]
+
+
+class varnttst_read_cmpx_var(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [
+            {
+                "id": 0,
+                "arr": ['"12, 22"', '"hello, hi, bye"'],
+                "mapp": {"a": '"ciao!"', "b": '"hello 45 234"'},
+                "roww": {"v1": '"20"', "v2": '"bye bye!"'},
+            },
+            {"id": 1, "arr": None, "mapp": None, "roww": None},
+            {
+                "id": 2,
+                "arr": [None, None],
+                "mapp": None,
```

Comment:
this field should exist if you use the right name.
please make sure when you check manually